### DISCUSSION TITLE
meeting layout changed

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1367] [Q&A] [answered]
Hi! I noticed today that the meeting layout has changed so that the info is justified left and squished (for lack of better words). I am not the only one that administers this site, so am not sure what was changed.

Do you have any ideas what could have caused this and how to fix it?

The page I need help with:  [https://rvaacadf.org/meetings/?meeting=freedom-from-the-past](url)

Thank you in advance for any thoughts/suggestions!

**Comment by joshreisner** [comment]
weird. we haven't made any changes in the last few months. maybe the theme updated?

anyway good timing, i was about to make a change to help another user deal with interference from their theme so i can add an extra `width: 100%` to the parent element which should fix it

**Reply by joshreisner** [reply, marked as answer]
ok! the [fix is out](https://github.com/code4recovery/tsml-ui/pull/363), if you refresh your browser window once or twice it should look correct

**Comment by gkovats** [comment]
@ACAwebbuilder closing this, your page looks with with the update / full page width.
